website/content/en/docs/advanced-topics/scorecard/kuttl-tests.md

```diff
@@ -4,160 +4,226 @@ weight: 50
+Let's say you've created an API kind `Memcached`, group `cache`, and version `v1alpha1`.
+The kuttl test files would look like:
+
+```console
+$ tree test/
+test/
+└── kuttl
+    ├── kuttl-test.yaml
+    └── v1alpha1
+        ├── 00-install-memcached.yaml
+        ├── 01-assert-memcached.yaml
+        └── 01-modify-memcached.yaml
+```
+
+Here's a quick overview of what the files do:
+- `test/kuttl/kuttl-test.yaml` - contains a kuttl [`TestSuite` configuration][suite].
+This particular configuration is geared towards running your test cases via scorecard so is quite minimal.
+- `test/kuttl/v1alpha1/00-install-memcached.yaml` - installs a Memcached CR with a minimal spec.
+- `test/kuttl/v1alpha1/01-modify-memcached.yaml` - updates the installed Memcached's spec.
+- `test/kuttl/v1alpha1/01-assert-memcached.yaml` - asserts that the spec update occurred.
+
+Lets take a look at a few of these files, first being `kuttl-test.yaml`:
+
+```yaml
+apiVersion: kudo.dev/v1beta1
+kind: TestSuite
+testDirs:
+- v1alpha1
+# +kubebuilder:scaffold:kuttl:testDirs
+timeout: 120
+```
+
+First, you should be aware that while this file looks like a `kubectl apply`-able manifest,
+it is in fact a component configuration file only meant to be read by kuttl commands. It uses
+Kubernetes versioning semantics to convey a defined degree of compatibility. This particular
+configuration includes a single test directory `v1alpha1`, the path to our test steps relative
+to the config file. The `# +kubebuilder:scaffold:kuttl:testDirs` comment helps `operator-sdk`
+commands update this file when new APIs are added, so you don't need to worry (or remove) it.
 
-## Defining kuttl Tests in Scorecard
+Next, if you peek inside `01-assert-memcached.yaml` you'll see something like the following:
 
-Scorecard users can include kuttl tests within their operator
-bundles as follows:
+```yaml
+apiVersion: cache.example.com/v1alpha1
+kind: Memcached
+metadata:
+  name: memcached-test
+spec:
+  foo: bar
 ```
-.
+
+Yep, it's a plain Memcached CR manifest, and it _is_ `kubectl apply`-able! Most if not all kuttl
+test case files are just Kubernetes YAML manifests that are either merge-patched against in-cluster
+objects or compared in assert or error scenarios. Hopefully you can see how powerful this
+declarative testing pattern can be.
+
+For further reading take a look through the kuttl [testing docs][kuttl-testing].
+
+## Running kuttl Test Suites with `scorecard`
+
+`operator-sdk scorecard` runs your kuttl suite by passing it to `kubectl-kuttl`, the kuttl testing CLI tool,
+from within the `quay.io/operator-framework/scorecard-test-kuttl` image. Since `scorecard` runs tests against
+a running operator using its [bundle][scorecard-bundle], your kuttl tests are expected to exist within either a
+[local bundle directory][doc-bundle] generated using `make bundle` or a bundle image. However, you
+should not bundle test dependencies with your production manifests. This problem is solved by
```

Comment:
why not? What is the problem with push the config.yaml with the bundle? 
What issue will it cause?